doc/MANUAL.md

```diff
@@ -241,61 +241,63 @@ which is useful if you're trying the latest code from
+| motion          | mp4v2         | mpg123             | mpv                | msmtp            | mtr             | mupdf         |
+| mutt            | mysql         | nano               | nasm               | nbd              | ncurses         | neon          |
+| nessus          | netatalk      | netdata            | netkit_ftp         | netpbm           | nettle          | nghttp2       |
+| nginx           | ngircd        | nmap               | node               | ntfs_3g          | ntp             | ntpsec        |
+| oath_toolkit    | open_iscsi    | open_vm_tools      | openafs            | opencv           | openjpeg        | openldap      |
+| opensc          | openssh       | openssl            | openswan           | openvpn          | orc             | p7zip         |
+| pango           | patch         | pcre               | pcre2              | pcsc_lite        | perl            | php           |
+| picocom         | pigz          | pixman             | png                | polarssl_fedora  | poppler         | postgresql    |
+| ppp             | privoxy       | procps_ng          | proftpd            | protobuf_c       | pspp            | pure_ftpd     |
+| putty           | python        | qemu               | qpdf               | qt               | quagga          | radare2       |
+| radvd           | raptor        | rauc               | rdesktop           | readline         | rpm             | rsync         |
+| rsyslog         | rtl_433       | rtmpdump           | runc               | rust             | samba           | sane_backends |
+| sdl             | seahorse      | shadowsocks_libev  | snapd              | sngrep           | snort           | socat         |
+| sofia_sip       | speex         | spice              | sqlite             | squashfs         | squid           | sslh          |
+| stellarium      | strongswan    | stunnel            | subversion         | sudo             | suricata        | sylpheed      |
+| syslogng        | sysstat       | systemd            | tar                | tcpdump          | tcpreplay       | terminology   |
+| tesseract       | thrift        | thttpd             | thunderbird        | timescaledb      | tinyproxy       | tor           |
+| tpm2_tss        | traceroute    | transmission       | trousers           | ttyd             | twonky_server   | u_boot        |
+| udisks          | unbound       | unixodbc           | upx                | util_linux       | varnish         | vim           |
+| vlc             | vorbis_tools  | vsftpd             | webkitgtk          | wget             | wireshark       | wolfssl       |
+| wpa_supplicant  | xerces        | xml2               | xscreensaver       | xwayland         | yasm            | zabbix        |
+| zchunk          | zeek          | zlib               | znc                | zsh              | zstandard       |               |
+
```

Comment:
One more comment, this change should be reverted because this table is built automatically by a tool so it will be overwritten anyway
You can see an example here: https://github.com/intel/cve-bin-tool/pull/4648